src/Compilers/Core/Portable/Symbols/ITypeParameterSymbol.cs

```diff
@@ -65,6 +65,11 @@ public interface ITypeParameterSymbol : ITypeSymbol
         /// </summary>
         ImmutableArray<ITypeSymbol> ConstraintTypes { get; }
 
+        /// <summary>
+        /// The nullability of the constraints on the type parameter.
+        /// </summary>
+        ImmutableArray<Nullability> ConstraintsNullability { get; }
```

Comment:
this name is inconsistent with ConstraintTypes.  it also doesn't seem to match the naming of anything else in the PR.